### ISSUE TITLE
Hadean Portals using incorrect effects

### ISSUE THREAD

**Issue by jack775544** [BlackDragonN001/FERemastered#99]
Hadean portals now have the incorrect effect while activated.
Will probably need a dll update to fix.

![20200516100508_1](https://user-images.githubusercontent.com/8287765/82105093-dbe19180-975c-11ea-8ede-252c28f33dc8.jpg)

**Comment by jack775544** [comment]
I have tracked down the cause for this one and it seems to be an issue with portals placed in older commits causing some bad data in the bzn.

To reproduce:
* Revert to [this](https://github.com/BlackDragonN001/FERemastered/commit/d607deb93730e428163ec3e2cb4ad41361a55048) commit (it is the one before the new portal model)
* Place a portal down on the map in noscript mode and save the map
* Write a lua script for the map that clears the portal destination
* Go back to master
* Load map and take a look

This behaviour does not occur if the portal was placed while on master.

I have uploaded my example map below.
[portals2.zip](https://github.com/BlackDragonN001/FERemastered/files/4643256/portals2.zip)
